public/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css" />
```

Comment:
Nick, I can see that you're using a CSS-only UI Framework in a React project. Can you explain why you're making this choice over tools such as [`semantic-ui-react`](https://react.semantic-ui.com/)?

Reply:
right now, I'm just using the css inline (something quick and easy to use). I don't know we are going to change to different framework, and prefer getting functionalities done first to design.

Reply:
Large projects whose key choices are provided through accretion are generally awful as they lead to heartaches much later on. Can you provide a better answer than "developer ergonomics" for a static header?